include/cbl/CBLDocument.h

```diff
@@ -76,10 +77,11 @@ const CBLDocument* _cbl_nullable CBLDatabase_GetDocument(const CBLDatabase* data
 CBL_REFCOUNTED(CBLDocument*, Document);
 
 /** Saves a (mutable) document to the database.
-    \warning If a newer revision has been saved since \p doc was loaded, it will be overwritten by
+    @warning If a newer revision has been saved since \p doc was loaded, it will be overwritten by
             this one. This can lead to data loss! To avoid this, call
             \ref CBLDatabase_SaveDocumentWithConcurrencyControl or
             \ref CBLDatabase_SaveDocumentWithConflictHandler instead.
+    @warning  Deprecated : Use CBLCollection_SaveDocument on the default collection instead.
```

Comment:
Have you checked the result to make sure two warnings shows up sanely?

Reply:
Yes, there will be grouped into the same warning section and they will display in different lines.